apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -86,6 +86,13 @@ $cyan-idle: #0aa;
       line-height: 0.8em;
     }
 
+    .studio-timer-seconds {
+      color: $red-active;
+      font-size: calc(#{$clock-size} / 4.6);
+      margin-top: calc(50% - calc(#{$clock-size} / 11));
+      line-height: 0.8em;
+    }
+
     .next-title:after,
     .next-title:after,
     .next-countdown__overtime:after {
```

Comment:
There is a typo here, not your fault. Would you mind fixing it?
it should read
`.next-countdown--overtime:after`